README.md

```diff
@@ -663,7 +663,8 @@ _Related: [Community-Supported Agriculture (CSA)](#community-supported-agricultu
 - [Pretix](https://pretix.eu/) - Django based ticket sales platform for events. ([Source Code](https://github.com/pretix)) `Apache-2.0` `Python`
 - [Reaction Commerce](https://reactioncommerce.com/) - Customizable, real-time reactive, Javascript commerce platform. ([Source Code](https://github.com/reactioncommerce/reaction)) `GPL-3.0` `Nodejs`
 - [Saleor](https://saleor.io) - Django based open-sourced e-commerce storefront. ([Demo](https://demo.getsaleor.com/), [Source Code](https://github.com/mirumee/saleor)) `BSD-3-Clause` `Python`
-- [Shopware Community Edition](https://shopware.com/community/) - PHP based open source e-commerce software made in Germany. ([Demo](https://www.shopware.com/en/test-demo/), [Source Code](https://github.com/shopware/platform)) `MIT` `PHP`
+- [Shopware 5 Community Edition](https://shopware.com/en/download/) - PHP based open source e-commerce software made in Germany. [Source Code](https://github.com/shopware/shopware)) `Open Source AGPL v3` `PHP`
```

Comment:
- invalid license tag
- only add a single entry regardless of the version number (only keep the shopware 6 entry)